extensions/applicationinsights-properties-js/Context/Device.ts

```diff
@@ -1,4 +1,4 @@
-﻿import { IDevice } from '../../JavaScriptSDK.Interfaces/Context/IDevice';
+﻿import { IDevice } from '../Interfaces/Context/IDevice';
```

Comment:
Sorry, meant to ask is Channel not using the same?

Reply:
No. Channel isn't aware of Part A fields. Since the system properties are handled here, the channel doesn't need to know about. 

---
In reply to: [214087308](https://github.com/Microsoft/ApplicationInsights-JS/pull/660#discussion_r214087308) [](ancestors = 214087308)